daemon/module/docker/remotevolumedriver/remvoldriver.go

```diff
@@ -112,8 +112,9 @@ var (
 )
 
 type pluginRequest struct {
```

Comment:
Hi @clintonskitson,

In your type definition there are two issues:

1. JSON fields should start with lowercase letters followed by camel case.
2. `ommitempty` should only have one `m`.

The type should be:

```
type pluginRequest struct {
    Name       string            `json:"name,omitempty"`
    Opts       volume.VolumeOpts `json:"opts,omitempty"`
    InstanceID string            `json:"instanceId,omitempty"`
}
```